templates/se2-k8s/.suborbital/se2-controlplane-deployment.yaml.tmpl

```diff
@@ -50,6 +62,18 @@ spec:
               value: "info"
             - name: SE2_CONTROL_PLANE
               value: "se2-controlplane-service:8081"
+          livenessProbe:
+            httpGet:
+              path: /api/v1/health
+              port: 8080
```

Comment:
This might not work due to the builder only responding to the healthcheck on the TLS port when `SE2_TLS_PORT` is set. Is that true @cohix?

Reply:
@arbourd I think we'll end up removing the `se2-k8s` template since that's for self-hosted.

Reply:
Ya this is mostly preparation for  https://github.com/suborbital/e2core/pull/339, so whatever `e2` is using.